lib/index.js

```diff
@@ -25,12 +25,14 @@ var MasonryComponent = React.createClass({
 
     propTypes: {
         disableImagesLoaded: React.PropTypes.bool,
+        onImagesLoaded: React.PropTypes.func,
         options: React.PropTypes.object
     },
 
     getDefaultProps: function() {
         return {
             disableImagesLoaded: false,
+            onImagesLoaded: _ => _,
```

Comment:
We can't use arrow functions in source just yet as we need to support older versions of node at this moment. 

If we setup a build step then sure, but right now we don't have one.